documentation/deliberately_vulnerable_flask_app/requirements.txt

```
@@ -0,0 +1,2 @@
+Flask
+pyre-check-nightly
```

Internal Lint complain about missing newline at the end
```suggestion
pyre-check-nightly

```